README.md

```diff
@@ -38,99 +38,103 @@ On Windows, download the installer [from GitHub](https://github.com/canonical/mu
 # Usage
 
 ## Find available images
-```
+```shell
 $ multipass find
 Image                       Aliases           Version          Description
-core                        core16            20200213         Ubuntu Core 16
-core18                                        20200210         Ubuntu Core 18
-16.04                       xenial            20200721         Ubuntu 16.04 LTS
-18.04                       bionic,lts        20200717         Ubuntu 18.04 LTS
-20.04                       focal             20200720         Ubuntu 20.04 LTS
-daily:20.10                 devel,groovy      20200721         Ubuntu 20.10
+core                        core16            20200818         Ubuntu Core 16
+core18                                        20211124         Ubuntu Core 18
+core20                                        20230119         Ubuntu Core 20
+core22                                        20230717         Ubuntu Core 22
+20.04                       focal             20240710         Ubuntu 20.04 LTS
+22.04                       jammy             20240716         Ubuntu 22.04 LTS
+24.04                       noble,lts         20240710         Ubuntu 24.04 LTS
+daily:24.10                 oracular,devel    20240719         Ubuntu 24.10
+appliance:adguard-home                        20200812         Ubuntu AdGuard Home Appliance
+appliance:mosquitto                           20200812         Ubuntu Mosquitto Appliance
+appliance:nextcloud                           20200812         Ubuntu Nextcloud Appliance
+appliance:openhab                             20200812         Ubuntu openHAB Home Appliance
+appliance:plexmediaserver                     20200812         Ubuntu Plex Media Server Appliance
 ```
 
 ## Launch a fresh instance of the current Ubuntu LTS
-```
-$ multipass launch ubuntu
-Launching dancing-chipmunk...
-Downloading Ubuntu 18.04 LTS..........
-Launched: dancing chipmunk
+```shell
+$ multipass launch --name vm-01 lts
+Launched: vm-01
 ```
 
 ## Check out the running instances
-```
+```shell
 $ multipass list
-Name                    State             IPv4             Release
-dancing-chipmunk        RUNNING           10.125.174.247   Ubuntu 18.04 LTS
-live-naiad              RUNNING           10.125.174.243   Ubuntu 18.04 LTS
-snapcraft-asciinema     STOPPED           --               Ubuntu Snapcraft builder for Core 18
+Name                    State             IPv4             Image
+vm-01                   Running           10.191.202.10    Ubuntu 24.04 LTS
```

Comment:
Let's keep the `multipass list` output populated with additional instances to show that Multipass can manage several instances for different proposes. For example, another running instance and a `charm-dev` machine.

Reply:
It looks unexpected. Especially with not explicitly provided names. Probably we need a separate step to run another machines?

Reply:
But anyway - I will add it. Just an outside view.

Reply:
I don't think we need to explicitly show the launching of the additional instances. Having multiple instances in the list output is mainly just to show that Multipass can handle the management of multiple instances.